### PR TITLE
Feature/twitter refresh

### PR DESCRIPTION
**Pull request by erik-sn**
This takes the already existing functionality of broadcasting tweets that was implemented for notifications and just prepends the tweet to the list of existing tweets. This means that the list of tweets on the phone should be updated in real time without making database calls.

I tested this as much as I could with two accounts connected to the same local server, I _assume_ it will work at a larger scale with many players.
[Feature/twitter refresh](https://app.gitkraken.com/glo/card/b0bc1bd7a3e64950b90089ae84d2173f)

### CODE REVIEW

**Review by jfrader — APPROVED** [review]

File: phone/src/apps/twitter/components/TwitterApp.tsx [hunk 1]
```diff
@@ -34,29 +34,13 @@ const useStyles = makeStyles(() => ({
...
-  useEffect(() => {
-
-    // this is a polling implementation. It is possible that
-    // there is some interaction where, on a new tweet, all
-    // clients are sent the updated query data. Until that can
-    // be accomplished this is naive but robust.
-    //
-    // TODO don't call fetchTweets - implement a function that only
-    // returns tweets that we don't already have
-    const timeout = window.setTimeout(() => {
```
[the remark this answers is not in the document]
Comment (RockySouthpaw) [reply phone/src/apps/twitter/components/TwitterApp.tsx:54]: It was working back in October as the teaser has it fully fetching properly lol

**Comment by RockySouthpaw** [comment]
**Issue**
- Tweets are fetching as if you're the one who tweeted them. **only when twitter is opened.**
     - This allows you the delete option and the other options such as retweet and report isnt there. It doesn't actually delete the other persons tweet though.

**Comment by erik-sn** [comment]
@RockySouthpaw I resolved the bug, was an oversight on my part.

**Review by TasoOneAsia — APPROVED** [review]

**Comment by RockySouthpaw** [comment]
Big is fixed.

**Review by RockySouthpaw — APPROVED** [review]

**Pull request merged**